inference-engine/tests/functional/plugin/cpu/single_layer_tests/interpolate.cpp

```diff
@@ -546,55 +641,85 @@ const std::vector<ShapeParams> shapeParams5D = {
     ShapeParams{
         ngraph::op::v4::Interpolate::ShapeCalcMode::SCALES,
         InputShape{{-1, {2, 20}, -1, -1, -1}, {{1, 11, 4, 4, 4}, {2, 7, 6, 5, 8}}},
-        ngraph::helpers::InputLayerType::CONSTANT,
-        {{1.f, 1.f, 1.25f, 1.5f, 0.5f}},
+        ngraph::helpers::InputLayerType::PARAMETER,
+        {{1.f, 1.f, 1.25f, 1.5f, 0.5f}, {1.f, 1.f, 1.25f, 1.25f, 1.25f}},
         defaultAxes5D.front()
     },
     ShapeParams{
         ngraph::op::v4::Interpolate::ShapeCalcMode::SIZES,
-        InputShape{{-1, {2, 20}, -1, -1, -1}, {{1, 11, 4, 4, 4}, {1, 11, 5, 5, 8}}},
-        ngraph::helpers::InputLayerType::CONSTANT,
-        {{1, 11, 5, 6, 4}},
+        InputShape{{-1, {2, 20}, -1, -1, -1}, {{1, 11, 4, 4, 4}, {2, 7, 6, 5, 8}}},
+        ngraph::helpers::InputLayerType::PARAMETER,
+        {{1, 11, 6, 7, 2}, {2, 7, 8, 7, 4}},
         defaultAxes5D.front()
     },
+};
+
+const std::vector<ShapeParams> shapeParams5D_Full = {
     ShapeParams{
         ngraph::op::v4::Interpolate::ShapeCalcMode::SCALES,
         InputShape{{-1, {2, 20}, -1, -1, -1}, {{1, 11, 4, 4, 4}, {2, 7, 6, 5, 8}}},
-        ngraph::helpers::InputLayerType::PARAMETER,
-        {{1.f, 1.f, 1.25f, 1.5f, 0.5f}, {1.f, 1.f, 1.25f, 1.25f, 1.25f}},
+        ngraph::helpers::InputLayerType::CONSTANT,
+        {{1.f, 1.f, 1.25f, 1.5f, 0.5f}},
         defaultAxes5D.front()
     },
     ShapeParams{
         ngraph::op::v4::Interpolate::ShapeCalcMode::SIZES,
-        InputShape{{-1, {2, 20}, -1, -1, -1}, {{1, 11, 4, 4, 4}, {2, 7, 6, 5, 8}}},
-        ngraph::helpers::InputLayerType::PARAMETER,
-        {{1, 11, 6, 7, 2}, {2, 7, 8, 7, 4}},
+        InputShape{{-1, {2, 20}, -1, -1, -1}, {{1, 11, 4, 4, 4}, {1, 11, 5, 5, 8}}},
+        ngraph::helpers::InputLayerType::CONSTANT,
+        {{1, 11, 5, 6, 4}},
         defaultAxes5D.front()
-    },
+    }
 };
 
-const auto interpolateCasesLinearOnnx5D = ::testing::Combine(
+const auto interpolateCasesLinearOnnx5D_Smoke = ::testing::Combine(
+        ::testing::Values(ngraph::op::v4::Interpolate::InterpolateMode::linear_onnx),
+        ::testing::ValuesIn(coordinateTransformModes_Smoke),
+        ::testing::ValuesIn(nearestModes_Smoke),
+        ::testing::ValuesIn(antialias),
+        ::testing::ValuesIn(pads5D),
+        ::testing::ValuesIn(pads5D),
+        ::testing::ValuesIn(cubeCoefs));
+const auto interpolateCasesLinearOnnx5D_Full = ::testing::Combine(
         ::testing::Values(ngraph::op::v4::Interpolate::InterpolateMode::linear_onnx),
-        ::testing::ValuesIn(coordinateTransformModes),
-        ::testing::ValuesIn(nearestModes),
+        ::testing::ValuesIn(coordinateTransformModes_Smoke),
+        ::testing::ValuesIn(nearestModes_Full),
```

Comment:
```suggestion
        ::testing::ValuesIn(defNearestModes),
```
nearest mode is used only for nearest neighbor mode. For other modes, just use defNearestModes as stub to not combine all nearest mode for other interpolate mode. Seems like it's not set properly since some time.

Reply:
Fixed in: 06054aceba7ccb9727eac838c9dd54798ca7b60e